src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -245,22 +182,50 @@ public function editAction($featureId, Request $request)
         ]);
     }
 
+    public function getLegacyControllerBridge(): LegacyControllerBridgeInterface
+    {
+        return $this->buildLegacyControllerBridge('feature', Feature::class, 'AdminFeatures');
+    }
+
     public function getHelperListBridge(): HelperListFeatureBridge
     {
         return $this->get('prestashop.core.bridge.helper_list_feature');
     }
 
-    private function buildGenericAction(): void
+    /**
+     * {@inheritdoc}
+     */
+    public function getIdentifier(): string
```

Comment:
I think we should also modify the `FrameworkBridgeListControllerInterface` in the same spirit we did for the `FrameworkBridgeControllerInterface` meaning it doesn't implement a method for each parameter that we need but instead it has to implement a `getHelperListConfiguration(ControllerConfiguration $controllerConfiguration)` and the trait contains a build method to help us build it (like for the controller configuration and controller bridge).

This way we limit the number of methods, and the interface represents more its purpose and not only the parameters that we need.